templates/Dockerfile.erb

```diff
@@ -47,6 +49,10 @@ RUN rm /usr/lib/ruby/gems/2.3.0/gems/facter-"$FACTER_VERSION"/lib/facter/blockde
 <% end %>
 <% end %>
 
+<% if use_factfile %>
+    COPY <%= factfile %> /etc/puppetlabs/facter/facts.d/custom_facts.txt
```

Comment:
Just wondering if we should give an option to the user where to copy the files
/opt/puppetlabs/facter/facts.d/
/etc/puppetlabs/facter/facts.d/
/etc/facter/facts.d/